lib/circuit.js

```diff
@@ -11,6 +11,7 @@ const CLOSED = Symbol('closed');
 const HALF_OPEN = Symbol('half-open');
 const PENDING_CLOSE = Symbol('pending-close');
 const FALLBACK_FUNCTION = Symbol('fallback');
+const FAILURE_CHECK_FUNCTION = Symbol('is-failure');
```

Comment:
Nit: the convention in `opossum` is that the `Symbol` name is the lowercase version of the `const` name, with `s/_/-/`.